<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Logining</class>
 <widget class="QWidget" name="Logining">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Min_pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/images/images/button-quit.svg); border:none;}

                                      QPushButton:hover{border-image: url(:/images/images/button-quit.svg);}
                                      QPushButton:pressed{border-image: url(:/images/images/button-quit.svg);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/button-minimize.svg</normaloff>:/images/images/button-minimize.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Close_pushButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>0</y>
     <width>20</width>
     <height>20</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image: url(:/images/images/button-quit.svg);border:none;}
                                      QPushButton:hover{border-image: url(:/images/images/button-quit.svg);}
                                      QPushButton:pressed{border-image: url(:/images/images/button-quit.svg);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/button-quit.svg</normaloff>:/images/images/button-quit.svg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="avatar_label">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>40</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff781f;&quot;&gt;头像&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/avatar.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="login_button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff781f;&quot;&gt;登录&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
    color: white;
    font: bold 13pt;
    border:none;
    min-height: 24px;
    min-width: 60px;
	border-image: url(:/images/images/button-login.svg);
}
QPushButton:hover{
    color: lightgray;
    
	border-image: url(:/images/images/button-login-hover.svg);
}
QPushButton:pressed {
    color: lightgray;
    
	border-image: url(:/images/images/button-login-press.svg);
    padding-top: -15px;
    padding-bottom: -17px;
} </string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>380</width>
     <height>5</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>正在加载</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QProgressBar {
	background-color: rgb(52, 187, 231);
    border: -1px solid;
    border-radius: 5px;
}

QProgressBar::chunk {
   
	background-color: rgb(255, 120, 31);
}</string>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentStatus_label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ff781f;&quot;&gt;正在链接服务器.......&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
